Gruntfile.js

```diff
@@ -97,7 +97,7 @@ const fetchLernaProjects = (log, runAllTests) => {
   // if JSON parse fails, well, grunt will just fail /shrug
   const parseLernaList = (cmd) => {
     try {
-      return JSON.parse(exec(`yarn -s lerna ${cmd} -a --no-ignore-changes --json --loglevel warn 2>&1`));
```

Comment:
@tjdett What is the effect of this change?

Reply:
When `cmd == 'list'` (i.e. `lerna list`) it will cause the command actually run, as opposed to exiting because `--no-ignore-changes` is an invalid option. This "invalid parameter" error was being masked by the `try` clause, and converted into an empty list.

There should not be a functional effect to the usual `lerna changed` usage, as the option is passed in along with the command.